molgenis-model-registry/src/test/java/org/molgenis/model/registry/controller/ModelRegistryControllerTest.java

```diff
@@ -0,0 +1,263 @@
+package org.molgenis.model.registry.controller;
+
+import com.google.common.collect.Lists;
+import org.molgenis.data.AbstractMolgenisSpringTest;
+import org.molgenis.data.MolgenisDataException;
+import org.molgenis.data.TestHarnessConfig;
+import org.molgenis.data.i18n.LanguageService;
+import org.molgenis.data.meta.MetaDataService;
+import org.molgenis.data.meta.model.Package;
+import org.molgenis.data.meta.model.PackageFactory;
+import org.molgenis.data.semantic.LabeledResource;
+import org.molgenis.data.semanticsearch.service.TagService;
+import org.molgenis.data.settings.AppSettings;
+import org.molgenis.framework.ui.MolgenisPluginRegistry;
+import org.molgenis.model.registry.model.*;
+import org.molgenis.model.registry.services.MetaDataSearchService;
+import org.molgenis.model.registry.services.TreeNodeService;
+import org.molgenis.model.registry.utils.ModelRegistryTestHarness;
+import org.molgenis.ui.menu.Menu;
+import org.molgenis.ui.menu.MenuReaderService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Import;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.ui.ExtendedModelMap;
+import org.testng.annotations.Test;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import static org.testng.Assert.assertEquals;
+
+/**
+ * <p>Test for <code>StandardRegistryTest</code>.</p>
```

Comment:
Comment is incorrect and can be removed since the class name indicates which class is being tested.